extensions/interactions/NumberWithUnits/directives/oppia-response-number-with-units.component.spec.ts

```diff
@@ -16,50 +16,60 @@
  * @fileoverview Unit tests for the NumberWithUnits response.
  */
 
-require(
-  'interactions/NumberWithUnits/directives/' +
-  'oppia-response-number-with-units.component.ts');
-import { importAllAngularServices } from 'tests/unit-test-utils.ajs';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HtmlEscaperService } from 'services/html-escaper.service';
+import { ResponseNumberWithUnitsComponent } from './oppia-response-number-with-units.component';
 
-describe('oppiaResponseNumberWithUnits', function() {
-  let ctrl = null;
+describe('ShortResponseRatioExpressionInput', () => {
+  let component: ResponseNumberWithUnitsComponent;
+  let fixture: ComponentFixture<ResponseNumberWithUnitsComponent>;
 
-  let mockHtmlEscaperService = {
-    escapedJsonToObj: function(answer) {
-      return answer;
+  class mockHtmlEscaperService {
```

Comment:
Class name should start with a capital letter.

Reply:
Done